Object[] obj_lst
---
Object[] obj_lst